crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -1918,15 +1920,44 @@ async fn test_object_no_id_error() {
                  && err_str.contains("First field of struct NotObject must be 'id'"));
 }
 
-pub async fn build_and_try_publish_test_package(
+#[tokio::test]
+#[cfg_attr(msim, ignore)]
+async fn test_publish_verification() {
```

Comment:
this test fails (that is succeeds publishing thus fail the test) without the change